src/main/kotlin/com/hibob/academy/employeeFeedback/resource/FeedbackResource.kt

```diff
@@ -57,4 +57,20 @@ class FeedbackResource(private val feedbackService: FeedbackService, private val
 
         return Response.ok(statusOfMyFeedback).build()
     }
+
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/feedbackId/{feedbackId}/markReviewed")
+    fun markReviewedO(@RequestBody reviewed: Boolean, @PathParam("feedbackId") feedbackId: Int, @Context request: ContainerRequestContext): Response {
```

Comment:
This is confusing a bit, because the path says markReviewed, so for me it means that it only changes the status to "reviewed"

but you are recieving a param, so it means that it can be both. Either create 2 endpoints, or change the url to clarify it

Reply:
fixed the url